fastlane/Fastfile

```diff
@@ -502,6 +507,25 @@ platform :ios do
     )
   end
 
+  private_lane :sign_xcframework do |options|
+    match(
+      readonly: true,
+      git_url: ENV['CERTS_REPO_URL'],
+      type: 'appstore',
+      skip_provisioning_profiles: true,
+      # This isn't important, we just need to fetch the certificate
+      app_identifier: "com.revenuecat.CarthageIntegration"
```

Comment:
BTW @joshdholtz how come this is required if we’re skipping provisioning profiles?